Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
+cni-sources:
+	@git clone https://github.com/aws/amazon-ecs-cni-plugins.git --branch $(revision)
+
+cni-plugins: cni-sources
```

Comment:
We could do this but I wanted to make building the cni-plugins as hassle free as possible. 
If we decouple these two targets we would end up with a fetch + build ideology.